src/components/vuestic-sass/icons/icons.scss

```diff
@@ -7,3 +7,5 @@ $ionicons-font-path: "~ionicons/dist/fonts";
 
 $fa-font-path: "~font-awesome/fonts";
 @import "~font-awesome/scss/font-awesome.scss";
+
+@import url(https://fonts.googleapis.com/icon?family=Material+Icons);
```

Comment:
@asvae According to design va-select va-file-upload va-sidebar-link-group use material-icons inside the vuestic-ui components. So, we need to support it right now to avoid errors in these components. 

Maybe it makes sense to move out investigation about it into separate issue

Reply:
Discussed with Vasily.

We'll keep material icons as main iconfont. Font-awesome support will go secondary.